### PR TITLE
[+] option not_save_tmpdir

### PR DESCRIPTION
**Pull request by ildarkhasanshin**

### CODE REVIEW

**Comment by Alexey-T** [comment]
@ildarkhasanshin I suggest more nice name for option- ignore_temp_files

File: __init__.py [hunk 1]
```diff
@@ -65,6 +68,10 @@ def save_one(e, msg):
     fn = e.get_filename()
     if not fn: return
 
+    if opt_not_save_tmpdir:
+        if fn.find(tempfile.gettempdir() + os.sep) != -1:
+            return
```
Comment (Alexey-T) [inline __init__.py:74]: @ildarkhasanshin why not to cache the temp-dir path to variable?

Comment (Alexey-T) [reply __init__.py:74]: @ildarkhasanshin why fn.find ? do it as `if fn.startswith()`

**Comment by Alexey-T** [comment]
@ildarkhasanshin See my comments on code, 2 cmts

**Comment by ildarkhasanshin** [comment]
* ignore_temp_files
* tmpdir
* fn.startswith(tmpdir)

is there anything else left?

**Comment by Alexey-T** [comment]
see above - you missed my comment on code. comments are duplicated in this discussion - above your last comment

**Comment by Alexey-T** [comment]
screenshot

di

**Pull request merged**